src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+import Big from 'big.js';
+
+const operations = {
+  '+': 'plus',
+  '-': 'minus',
+  '×': 'times',
+  '÷': 'div',
```

Comment:
Dont forget to manage division by ZERO when using this component

Reply:
I will, thanks!